modules/partition-replicator/src/test/java/org/apache/ignite/internal/partition/replicator/ZoneResourcesManagerTest.java

```diff
@@ -150,7 +158,15 @@ private static void assertThatStorageIsStopped(ZonePartitionResources resources)
         );
     }
 
-    private ZonePartitionResources allocatePartitionResources(ZonePartitionId zonePartitionId, int partitionCount) {
-        return bypassingThreadAssertions(() -> manager.allocateZonePartitionResources(zonePartitionId, partitionCount));
+    private ZonePartitionResources allocatePartitionResources(
+            ZonePartitionId zonePartitionId,
+            int partitionCount,
+            PendingComparableValuesTracker<Long, Void> storageIndexTracker
```

Comment:
You always pass the same value you have already stored in a field, so this parameter is redundant

Reply:
Same as above. It'll be covered in https://issues.apache.org/jira/browse/IGNITE-24654